src/stories/Grid.mdx

```diff
@@ -33,7 +33,7 @@ import * as GridStories from './Grid.stories.js';
 @use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/grid.scss';
 
 .your-css-selector {
-  @include grid.grid($align-left: false, $no-max: false);
+  @include grid.grid($align-left: false, $no-max: false, $no-pad: false);
```

Comment:
I think `$no-pad` here was meant to be `$no-gap` now. Everything else looks good to me!

Reply:
Sweet! This is fixed.